includes/definitions/eltek.yaml

```diff
@@ -0,0 +1,55 @@
+os: eltek
+text: 'Eltek Valere'
+type: power
+icon: eltek
+#sysObjectID: .1.3.6.1.4.1.12148
+nobulk: 1
+mib_dir:
+    - eltek
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    - .1.3.6.1.4.1.12148.10.2.4.0
```

Comment:
This isn't valid.  remove the discovery section.

Reply:
Gone, but now I don't understand how it is doing discovery!

Reply:
check #7558